apps/expo/app/index.tsx

```diff
@@ -1,21 +1,23 @@
 import React from "react";
 import { Button, Text, TextInput, TouchableOpacity, View } from "react-native";
 import { SafeAreaView } from "react-native-safe-area-context";
+import { Stack, useRouter } from "expo-router";
 import { FlashList } from "@shopify/flash-list";
 
-import { api, type RouterOutputs } from "../utils/api";
+import { api, type RouterOutputs } from "../src/utils/api";
 
 const PostCard: React.FC<{
   post: RouterOutputs["post"]["all"][number];
-  onPress: () => void;
   onDelete: () => void;
-}> = ({ post, onPress, onDelete }) => {
+}> = ({ post, onDelete }) => {
+  const router = useRouter();
+
   return (
     <View className="flex flex-row rounded-lg bg-white/10 p-4">
       <View className="flex-grow">
-        <TouchableOpacity onPress={onPress}>
+        <TouchableOpacity onPress={() => router.push(`/post/${post.id}`)}>
```

Comment:
would have liked a `<Link href="">` here but that seems to apply some nasty styles which i cba fixing so just used `router.push` instead :/